estela-web/src/components/JobDetailPage/index.tsx

```diff
@@ -1035,8 +1035,15 @@ export class JobDetailPage extends Component<RouteComponentProps<RouteParams>, J
                             <Col>
                                 <Text className="font-bold">Data Persistence</Text>
                             </Col>
-                            {dataStatus == "PENDING" && <Col className="col-span-2 px-2">{dataExpiryDays} days</Col>}
-                            {dataStatus == "PERSISTENT" && <Col className="col-span-2 px-2">Forever</Col>}
+                            <Col className="col-span-2 px-2">
+                                {dataStatus == "PENDING" ? (
+                                    `${dataExpiryDays} days`
+                                ) : dataStatus == "PERSISTENT" ? (
+                                    "Forever"
+                                ) : (
+                                    <p className="text-estela-red-full">Data deleted automatically</p>
```

Comment:
Please change this to `Job data was automatically deleted`.

Reply:
Done